### DISCUSSION TITLE
example of setting 5 screws bed leveling?

### DISCUSSION THREAD

**Discussion by Dweinbach** [KlipperScreen/KlipperScreen#1081] [Q&A]
Hi: I've been using klipper less than a week, and I'm trying to set up 5 screws leveling on klipperscreen. I've tried all kinds of variants of adding screw positions in KlipperScreen.conf in the printer section:
`screw_positions: "fl", "fr", "br", "bl", "center"`
`screw_positions: "fl, fr, br, bl, center"`
`screw_positions: fl,fr,br,bl,center`
but I always get:
`Bed screw configuration: 5
Not supported for auto-detection, it needs to be configured in klipperscreen.conf`
is there a trick? do I have the file name wrong (caps)?
which format is the row is correct, and does it belong in the printer section?

Thanks!

**Comment by alfrix** [comment]
to see what is being read you need to check the logfile, it will dump the configuration at each the start 
https://klipperscreen.readthedocs.io/en/latest/Troubleshooting/
the last one should have worked, and yes it goes in the printer section, you need to check the log

**Comment by Dweinbach** [comment]
I figured it out.  Operator error. I had added the srew_positions line to the END of the file, and hadn't noticed the "#~# --- Do not edit below this line. This section is auto generated --- #~#" above it - so it was being treated as a comment even though it didn't start with #
